website/docs/faq.md

```diff
@@ -41,6 +41,15 @@ You need to have a [Billing Manager](organization.md/#billing-manager-role) role
 
 This is possible if you join our partnership program or via [referrals](https://configcat.com/referrals/). [Contact us](https://configcat.com/support/) directly for more information.
 
+### Can I change the email address associated with my account?
+
+Currently, there's no direct way to change the email address of an account. However, you can:
+
+- Invite the New Email: Add the new email address to your account.
```

Comment:
feel free to rephrase, but this is the proper flow:

1. invite the new email to your CC organization, a new CC account will be created. 
2. grant all necessary permissions to the new accunt
3. deletete the old one (link)